sdk/ml/azure-ai-ml/azure/ai/ml/constants/_common.py

```diff
@@ -836,6 +836,13 @@ class Boolean:
     """False boolean type."""
 
 
+class InferenceServerType:
```

Comment:
declare as str + enum? 
Do you want it to be caseinsensitivenummeta too?

Reply:
Yes declaring the case sensitive way, as it is done for other similar areas